Bottom: da6a492fdf339e59c3d88071355d47dbfc83afc0
Top:    b67d2ba496b6fdfa009ab69e19f90349fadf0513
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 14:27:30 -0500

Add variable file for HSM host configuration


---
diff --git a/ansible-playbook/group_vars/buildbot/hsm.yml b/ansible-playbook/group_vars/buildbot/hsm.yml
new file mode 100644
index 00000000..aa3e1150
--- /dev/null
+++ b/ansible-playbook/group_vars/buildbot/hsm.yml
@@ -0,0 +1,2 @@
+---
+hsm_host: "172.16.64.38"
